lib/aptible/cli/helpers/tunnel.rb

```diff
@@ -0,0 +1,76 @@
+require 'socket'
+require 'open3'
+
+module Aptible
+  module CLI
+    module Helpers
+      class Tunnel
+        def initialize(env, cmd)
+          @env = env
+          @cmd = cmd
+        end
+
+        def start(desired_port = 0, err_fd = $stderr)
+          @local_port = desired_port
+          @local_port = random_local_port if @local_port == 0
+
+          # First, grab a remote port
+          out, err, status = Open3.capture3(@env, *@cmd)
+          fail "Failed to request remote port: #{err}" unless status.success?
```

Comment:
ssh on windows (msysgit binary) exists 1 on this command AND gives a valid port in `out`. 

Not exactly sure why...

I used `unless status.success? || out =~ /^[0-9]{3,}$/` to bypass